### ISSUE TITLE
Use composition instead of inheritence

### ISSUE THREAD

**Issue by timscott** [jagthedrummer/so_auth#4]
Maybe it would be better to include a module in the client's `ApplicationController` instead of requiring to inherit a base class? The client could include it explicitly or the gem might just do it automatically. I would be willing to make this change. What do you think?

On a side note, I would probably remove `protect_from_forgery`. Although I would never go without it, I'd prefer for it not to be hidden in a in mostly unrelated gem.

**Comment by jagthedrummer** [comment]
Yeah, I agree that including a module would be a little nicer than inheriting from a base class.
